chord_metadata_service/mohpackets/schemas/filter.py

```diff
@@ -88,38 +86,24 @@ class TreatmentFilterSchema(FilterSchema):
     submitter_primary_diagnosis_id: Optional[str] = Field(None)
     treatment_type: List[str] = Field(None, q="treatment_type__overlap")
     is_primary_treatment: Optional[str] = Field(None)
-    line_of_treatment: Optional[int] = Field(None)
-    treatment_setting: Optional[str] = Field(None)
     treatment_intent: Optional[str] = Field(None)
-    days_per_cycle: Optional[int] = Field(None)
-    number_of_cycles: Optional[int] = Field(None)
     response_to_treatment_criteria_method: Optional[str] = Field(None)
     response_to_treatment: Optional[str] = Field(None)
     status_of_treatment: Optional[str] = Field(None)
 
 
-class ChemotherapyFilterSchema(FilterSchema):
-    program_id: Optional[str] = Field(None)
-    submitter_donor_id: Optional[str] = Field(None)
-    submitter_treatment_id: Optional[str] = Field(None)
-    drug_reference_database: Optional[str] = Field(None)
-    drug_name: Optional[str] = Field(None)
-    drug_reference_identifier: Optional[str] = Field(None)
-    chemotherapy_drug_dose_units: Optional[str] = Field(None)
-    prescribed_cumulative_drug_dose: Optional[int] = Field(None)
-    actual_cumulative_drug_dose: Optional[int] = Field(None)
-
-
-class HormoneTherapyFilterSchema(FilterSchema):
+class SystemicTherapyFilterSchema(FilterSchema):
     program_id: Optional[str] = Field(None)
     submitter_donor_id: Optional[str] = Field(None)
     submitter_treatment_id: Optional[str] = Field(None)
     drug_reference_database: Optional[str] = Field(None)
     drug_name: Optional[str] = Field(None)
     drug_reference_identifier: Optional[str] = Field(None)
-    hormone_drug_dose_units: Optional[str] = Field(None)
+    drug_dose_units: Optional[str] = Field(None)
     prescribed_cumulative_drug_dose: Optional[int] = Field(None)
     actual_cumulative_drug_dose: Optional[int] = Field(None)
+    days_per_cycle: Optional[int] = Field(None)
+    number_of_cycles: Optional[int] = Field(None)
 
```

Comment:
Missing systemic_therapy_type? Feels like the sort of thing we should be able to filter on.